### DISCUSSION TITLE
ASA Add german Server-Page for Voting-System

### DISCUSSION THREAD

**Discussion by PlayButtonYT** [gameserverapp/Platform#602] [Suggestions]
Hello, I kindly request to include the following German ARK server page in the voting system. Especially for German players, this is actually the main website where players can search for and find servers. I've been using the site for my ARK SE servers for years now. Of course, the site also has APIs for the servers and works very reliably with ARK SE. Please consider adding it to the German community. This would be very helpful for us.

https://deutsche-arkserver.de/

**Comment by PlayButtonYT** [comment]
https://acserverlist.docs.apiary.io/#
API Doc´s

**Comment by PlayButtonYT** [comment]
https://deutsche-arkserver.de/api/v2/482B945C-EB37-4064-9B4F-4EF676B5DBF5/server/45152/
Doc´s

**Comment by mvaessen** [comment]
hi thanks, looking at the docs I probably wont be able to add this site because it doesn't have the required api endpoints. Please ask the vote site to implement similar API endpoints as like :https://ark-servers.net/help/api/

The important missing endpoint is for claiming a vote for a specific steam id. 
![Screenshot 2023-11-29 at 12 13 57](https://github.com/gameserverapp/Platform/assets/13402423/5f294d41-9b62-4209-8935-90b09d3557f9)


Let me know when they updated their API

**Reply by PlayButtonYT** [reply]
Okay, I've contacted the website. Let's see what they have to say about it. It's actually true that they require a separate registration and don't use Steam accounts. Nevertheless, I asked them politely. Maybe it will work out. In principle, it would be a huge advantage for both platforms because, as I mentioned, the German community primarily looks for servers on deutsche-arkserver.de.

**Reply by Sir-Will** [reply]
Hi @mvaessen, we do already have an endpoint for setting votes as claimed. Either through the batch update request or using the "VoteRewards Plugin" endpoint.

It's documented here:
https://acserverlist.docs.apiary.io/#reference/0/voterewards-plugin/get-unclaimed-votes-count

**Comment by mvaessen** [comment]
Please let me know when the site has replied with the required info. Closing for now.

**Comment by Dalo1101** [comment]
I have posted a suitable article on this in their forum. I would ask you to leave a comment there so that the operators see that there is interest

https://arkforum.de/thread/31265-api-anpassung-für-gsa/

**Comment by mvaessen** [comment]
The Deutsche-arkserver.de website should now be available as a vote site.

Use the image below to find your `API key` and `server id`.

![image](https://github.com/user-attachments/assets/e00289f5-b1ee-4ef7-9ee0-9c86a14397b9)

**Comment by Dalo1101** [comment]
Thanks a Lot Max 🙏
